src/cinder/volume/drivers/datera/datera_api22.py

```diff
@@ -1164,9 +1171,9 @@ def _support_template_override_2_2(self):
         # instantiation.
         if not self.template_override:
             return False
-        if hasattr(self, '_to_22'):
-            return self._to_22
-        api = self.api.api.get()
-        prop = api['/app_instances']['create']['bodyParamSchema']['properties']
-        self._to_22 = 'template_override' in prop
+        if not hasattr(self, '_to_22'):
+            api = self.api.api.get()
+            prop = api['/app_instances']['create']['bodyParamSchema'][
+                'properties']
+            self._to_22 = 'template_override' in prop
```

Comment:
This one. Where is exactly coming from?

Reply:
I am not sure about this piece of code. I added @NaveenSreeramachandra in case he know what's the history behind it.